ci/build_scripts/build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash -x
+
+set -euo pipefail
+
+help() {
+  cat <<EOF
+Compile and build the tedge components and debian packages.
```

Comment:
```suggestion
Compile and build the Debian packages for the tedge components.
```

Reply:
Though the script is creating both the raw binaries and then packing them as debian packages, so it is producing two artifacts...and moving forward we might use the binaries directly in some cases rather than via the debian package.